include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -862,6 +862,20 @@ class LAPACKFullMatrix : public TransposeTable<number>
   number
   singular_value(const size_type i) const;
 
+  /**
+   * Retrieves #svd_u matrix after compute_svd() or compute_inverse_svd() was
+   * called.
+   */
+  inline LAPACKFullMatrix<number>
```

Comment:
are you sure you want to return the matrix by value? I think a ``const &`` would be more appropriate,

Reply:
yes, you are right. I will change it to a `const &`